src/diff.c

```diff
@@ -1922,6 +1927,298 @@ diff_clear(tabpage_T *tp)
     tp->tp_first_diff = NULL;
 }
 
+/*
+ *  return true if the options are set to use diff linematch
+ */
+    Bool
+diff_linematch(diff_T *dp)
+{
+    if (!(diff_flags & DIFF_LINEMATCH)) {
+	return 0;
+    }
+    // are there more than three diff buffers?
+    int tsize = 0;
+    for (int i = 0; i < DB_COUNT; i++) {
```

Comment:
Can you move the opening curly brace for the "for", "if", "while", etc., statements to a separate line?

Reply:
moved to seperate lines